### ISSUE TITLE
Question on Diffusion Models images

### ISSUE THREAD

**Issue by davide-coccomini** [vishal3477/Reverse_Engineering_GMs#18]
Hi, thank you for the very useful dataset.
I have a question on the DMs part. I downloaded the dataset and I see that there are 8 folders called Z_i. What are the methods used to generate the images in each folder?

**Comment by vishal3477** [comment]
Hi,
Thank you for your interest in our work. The information for all models is given in the supplementary file of the paper.

**Comment by davide-coccomini** [comment]
Just to be sure, you are referring to Table 6? So the correspondences are:
Z_1 --> ADM
Z_2 --> ADM-G
Z_3 --> DDPM
Z_4 --> DDIM
Z_5 --> LDM
Z_6 --> Stable Diffusion (which version?)
Z_7 --> GLIDE

**Comment by davide-coccomini** [comment]
Can you confirm that?

**Comment by vishal3477** [comment]
Yes that is correct. Really sorry to reply so late. Please reach me directly on my email: asnanivi@msu.edu for a much faster response.

**Comment by davide-coccomini** [comment]
Thank you so much!
